### PR TITLE
Remove cookie env

### PR DESCRIPTION
**Pull request by JMurphyWeb**
want to minimise environment variables required in the project

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+dependencies:
```
Comment (nelsonic) [inline circle.yml]: are we still _using_ Circle-CI ?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon server/start.js",
-    "test": "node_modules/.bin/tape test/*.test.js",
+    "test": "node test",
```
Comment (nelsonic) [inline package.json]: this is only running _some_ of the tests  (_hence the coverage going down precipitously..._)

File: server/handlers.js [hunk 3]
```diff
@@ -1,87 +1,203 @@
...
+handlers.createNewOrganisation = (request, reply) => {
+  const redis = request.redis;
+  redis.LLEN('organisations', (error, length) => {
+    if (error) {
+      console.log(error);
+      reply(Boom.badImplementation('redis-failure'));
+    } else {
+      const orgUpdated = initialiseEntry(length, { name: request.payload.name, mission_statement: '', people: [] });
+      redis.RPUSH('organisations', orgUpdated, (error, numberOfOrgs) => {
+        if (error) {
```
Comment (nelsonic) [inline server/handlers.js]: This is a _good_ candidate for: https://github.com/dwyl/hapi-error :wink:

Comment (JMurphyWeb) [inline server/handlers.js]: Absolutely - error handling is a big something.

File: package.json (hunk 2 above)
Comment (JMurphyWeb) [inline package.json]: I am debugging travis and it is very helpful to only run one file at a time

**Comment by JMurphyWeb** [comment]
closing as we are hoping to use auth jwt2

**Pull request closed without merging**